core/src/main/java/com/backblaze/b2/json/B2Json.java

```diff
@@ -142,16 +144,20 @@ public static B2Json get() {
 
     /**
      * Turn an object into JSON, writing the results to given output stream.
+     *
+     * (package access for testing)
      */
-    public void toJson(Object obj, OutputStream out) throws IOException, B2JsonException {
+    void toJson(Object obj, OutputStream out) throws IOException, B2JsonException {
```

Comment:
why would we _remove_ "public" for testing purposes?  i can see removing "private", but I think this change is altering the SDK's API.

Reply:
same question for the other instances below.

Reply:
So far the methods with the public removed are used for testing only. I have not found that they are used as API. Another reason for making them non-public is to prevent the caller using the OOM-prone OutputStream in making these API calls.